addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -59,7 +59,8 @@ export class MassMailingHtmlField extends HtmlField {
             snippets: 'mass_mailing.email_designer_snippets',
             resizable: false,
             defaultDataForLinkTools: { isNewWindow: true },
-            toolbarTemplate: 'mass_mailing.web_editor_toolbar',
+            toolbarTemplate: device.isMobile ? 'web_editor.toolbar' : 'mass_mailing.web_editor_toolbar',
```

Comment:
Ah, this already prevents folding/unfolding.. Maybe use web_editor_toolbar everywhere? It might be ok